vertical-pod-autoscaler/pkg/admission-controller/logic/recommendation_provider_test.go

```diff
@@ -65,6 +65,9 @@ func TestUpdateResourceRequests(t *testing.T) {
 	vpaWithEmptyRecommendation := vpaBuilder.Get()
 	vpaWithEmptyRecommendation.Status.Recommendation = &vpa_types.RecommendedPodResources{}
 
+	checkLimit := true
+	setMemoryLimit = &checkLimit
```

Comment:
Do we have a tests that checks what happens if flag is false?

Reply:
Test case added.